### ISSUE TITLE
Problem when using society vehicles.

### ISSUE THREAD

**Issue by MrHtgreen** [ESX-Org/esx_mechanicjob#69]
I use society vehicles on the taxi job and they work fine but with this job even though the database is all fine I cant get past this error that comes up in the console when trying to access the vehicles menu in los santos.

SCRIPT ERROR: @esx_society/server/main.lua:329: attempt to index a nil value (local 'store')
> ref (@esx_society/server/main.lua:329)
> handler (@esx_datastore/server/main.lua:80)
> ref (@esx_society/server/main.lua:328)
> TriggerServerCallback (@es_extended/server/functions.lua:33)
> handler (@es_extended/server/common.lua:88)

Hope this can be fixed, thanks

**Comment by kekev2001** [comment]
SCRIPT ERROR: @esx_society/server/main.lua:329: attempt to index a nil value (local 'store')



> ref (@esx_society/server/main.lua:329)
> handler (@esx_datastore/server/main.lua:80)
> ref (@esx_society/server/main.lua:328)
> TriggerServerCallback (@es_extended/server/functions.lua:33)
> handler (@es_extended/server/common.lua:89)

**Comment by pavpd** [comment]
Yup that happens, just disable society owned vehicles in the config. It'd be good to be fixed, though

**Comment by PavlinHub** [comment]
Found a fix, make sure you have esx_mechanicjob in all: addon_account_data, addon_inventory, datastore, datastore_data.

**Comment by TOMMASO26175** [comment]
> Found a fix, make sure you have esx_mechanicjob in all: addon_account_data, addon_inventory, datastore, datastore_data.
yeah now it works just add into datastore
thx
